decksite/achievements.py

```diff
@@ -103,6 +102,14 @@ def load_summary(self, season_id: Optional[int] = None) -> Optional[str]:
             players_text = ngettext('1 player', f'%(num)d players', res.pnum)
             return f'Earned{times_text} by {players_text}.'
         return None
+    def percent(self, season_id: Optional[int] = None) -> float:
+        season_condition = f'season_id = {season_id}' if season_id != 'all' else ''
+        sql = f"""SELECT SUM(CASE WHEN {self.key} > 0 THEN 1 ELSE 0 END) AS pnum, COUNT(*) AS mnum FROM _achievements WHERE {season_condition}"""
```

Comment:
nit. Triple quotes for multiline strings, single quotes for single-line strings, double quotes for single-line strings containing apostrophes.